bin/job_ar.py

```diff
@@ -117,7 +117,7 @@ def main(args=None):
 
     # Command to clean a/r data from mongo
     cmd_clean_mongo_ar = [os.path.join(
-        stdl_exec, "mongo_clean_ar.py"), '-d', args.date, '-p', json_cfg["aprofile"]]
+        stdl_exec, "mongo_clean_ar.py"), '-d', args.date, '-r', json_cfg["job"], '-t', args.tenant]
```

Comment:
@pkoro, actually it does work because the field in the _cfg.json filed is declared as "job"
for eg in [EGI_Cloudmon_cfg.json](https://github.com/ARGOeu/argo-compute-engine/blob/ARGO-146-Multitenancy/conf/EGI_Cloudmon_cfg.json) 
```json

  "tenant":"EGI",
  "job":"Cloud",
  "egroup":"SITES",
  "ggroup":"NGI",
```

We could change it both here and in config files for consistency